todo_app/app.py

```diff
@@ -1,23 +1,66 @@
-from flask import Flask, request, redirect, render_template
+from flask import Flask, request, redirect, render_template, url_for
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import get_items, add_item, delete_item, completed_item, not_started_item
+
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = get_items()
-    #print(items)
+    items = get_items(app.config["BOARD_ID"],app.config["API_KEY"],app.config["API_TOKEN"])
```

Comment:
You're passing through the app config in lots of places, which feels slightly painful - if Trello starts to need another value we'd have to change a lot of code. Even a small change like passing through the entire `app.config` class and unpacking it in the method might make more sense

Reply:
Thanks, I see what you mean and I'll implement that